### DISCUSSION TITLE
Playlist.title

### DISCUSSION THREAD

**Discussion by taNtr1k** [Androz2091/discord-player#616] [Q&A]
Playlist.title does this exist if yes its not working when i am adding Spotify playlist

**Comment by twlite** [comment]
There is not a defined playlist object in v4 so we cannot say what exactly is the playlist object. However, in the next version we have added a fixed playlist object to solve this issue.

**Comment by taNtr1k** [comment]
Where can i find all the existing objects to develop my own code?

**Comment by twlite** [comment]
you can console log playlist object
